### PR TITLE
Inbound transfer API integration test added

### PR DESCRIPTION
**Pull request by danishjamal104**
Updates in this PR
1. New test case for inbound transfer API or `/channel/transfer` endpoint.
2. Added new config named `ChannelConnectorConfig.java`, for the configurations like contactpoint and endpoints related to channel connector. 

### Another PR needed to be reviewed and merged before this PR: https://github.com/openMF/ph-ee-connector-integration-test/pull/2

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,12 +10,18 @@ sourceCompatibility = '17'
 
 repositories {
 	mavenCentral()
+
+	maven {
+		url = uri('https://fynarfin.jfrog.io/artifactory/fyn-libs-snapshot-local/')
```
Comment (fynmanoj) [inline build.gradle:15]: is this publically open to reading?

File: src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        assertThat(mockTransactionChannelRequestDTO).isNotNull();
+    }
+
+    @When("I call the inbound transfer endpoint with expected status of {int}")
+    public void callBatchSummaryAPI(int expectedStatus) {
+        RequestSpecification requestSpec = Utils.getDefaultSpec(BaseStepDef.tenant);
+
+        BaseStepDef.response = RestAssured.given(requestSpec)
+                .baseUri(channelConnectorConfig.channelConnectorContactPoint)
+                .body(mockTransactionChannelRequestDTO) // todo check if this is correct?
```
Comment (fynmanoj) [inline src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java:55]: is that TODO done?  remove comment

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java (hunk 2 above)
Comment (danishjamal104) [reply src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java:55]: yes its done

File: src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                .expect()
+                .spec(new ResponseSpecBuilder().expectStatusCode(expectedStatus).build())
+                .when()
+                .post(channelConnectorConfig.transferEndpoint)
+                .andReturn().asString();
+
+        logger.info("Inbound transfer Response: " + BaseStepDef.response);
```
[the remark this answers is not in the document]
Comment (danishjamal104) [reply src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java:62]: ok

File: build.gradle (hunk 1 above)
Comment (danishjamal104) [reply build.gradle:15]: I am not aware of

File: src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (danishjamal104) [reply src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java]: done

File: src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+    public static TransactionChannelRequestDTO mockTransactionChannelRequestDTO = null;
+
+    @Given("I can mock TransactionChannelRequestDTO")
+    public void mockTransactionChannelRequestDTO() throws JsonProcessingException {
+        if (mockTransactionChannelRequestDTO != null) {
+            assertThat(mockTransactionChannelRequestDTO).isNotNull();
+            return;
+        }
+        String json = "{\n" +
```
[the remark this answers is not in the document]
Comment (danishjamal104) [reply src/test/java/org/mifos/integrationtest/cucumber/InboundStepDef.java]: updated

File: build.gradle (hunk 1 above)
Comment (danishjamal104) [reply build.gradle:15]: I think yes

**Review by fynmanoj — CHANGES_REQUESTED** [review]
can you remove cucumber report from the code

**Comment by danishjamal104** [comment]
> can you remove cucumber report from the code

done

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**